tests/DCPS/Compiler/rapidjson_generator/Main.cpp

```diff
@@ -0,0 +1,7 @@
+#include "gtest/gtest.h"
+
+int main(int argc, char* argv[])
```

Comment:
This test needs to have a `run_test.pl` and an entry in `dcps_tests.lst`

Reply:
Yes, I have a `run_test.pl` ready, I just need to figure out if it needs any flags for `dcps_tests.lst`